tdvt/tdvt/exprtests/standard/setup.string.endswith.null.txt

```diff
@@ -0,0 +1,3 @@
+ENDSWITH(str2, Null)
+ENDSWITH(str1,'')
+ENDSWITH(Num0, Null)
```

Comment:
Some of the col names start with an uppercase letter, others with lowercase. In tests/datasets/TestV1/postgres/load_calcs.sql I see that the cols are all named using lowercase. Were these conventions in the tests on the monolith? You could try changing them to lowercase (in this file and others) and then doing the proof run.